docs/design/.pages

```diff
@@ -1,5 +1,5 @@
 nav:
-    - ソフトウェアアーキテクチャ: design.md
-    - トレースポイントの定義: tracepoint_definition.md
+    - Architecture overview: architecture_overview.md
+    - Supported tracepoints: supported_tracepoint.md
```

Comment:
supported_tracepoint**s**.md

Reply:
Fixed in https://github.com/tier4/CARET_doc/pull/40/commits/e70eda1881e8fc8214f745c3bda1aeea170ae96b